server/cluster/coordinator.go

```diff
@@ -108,6 +110,10 @@ func (c *coordinator) patrolRegions() {
 			return
 		}
 
+		// Check miss region first
+		if c.checkMissRegions() {
```

Comment:
What if there are some region cannot be repaired? Will it block the other checkers?

Reply:
it need to manual if the situation happened